src/EnergyPlus/WaterToAirHeatPumpSimple.cc

```diff
@@ -1415,25 +1507,59 @@ namespace WaterToAirHeatPumpSimple {
+                                                               "COIL:" + simpleWatertoAirHP.WatertoAirHPType + ":WATERTOAIRHEATPUMP:EQUATIONFIT",
+                                                               simpleWatertoAirHP.Name,
+                                                               simpleWatertoAirHP.WaterInletNodeNum,
+                                                               simpleWatertoAirHP.WaterOutletNodeNum,
+                                                               ErrorsFound,
+                                                               false);
+                        if (PltSizNum > 0) {
+                            DesignEntWaterTemp = state.dataSize->PlantSizData(PltSizNum).ExitTemp;
+                            ratioTS = (DesignEntWaterTemp + state.dataWaterToAirHeatPumpSimple->CelsiustoKelvin) / Tref;
+                        } else {
+                            ShowSevereError(state, "Autosizing of total cooling capacity requires a loop Sizing:Plant object");
+                            ShowContinueError(state, "Autosizing also requires physical connection to a plant or condenser loop.");
+                            ShowContinueError(state,
+                                              "Occurs in COIL:" + simpleWatertoAirHP.WatertoAirHPType +
+                                                  ":WATERTOAIRHEATPUMP:EQUATIONFIT Object=" + simpleWatertoAirHP.Name);
+                            ErrorsFound = true;
+                        }
+                        // calculate temperatue ratio at rated conditions
+                        RatedratioTWB = (RatedMixWetBulb + state.dataWaterToAirHeatPumpSimple->CelsiustoKelvin) / Tref;
+                        RatedratioTS = (simpleWatertoAirHP.RatedEntWaterTemp + state.dataWaterToAirHeatPumpSimple->CelsiustoKelvin) / Tref;
+                        // determine curve modifiers at peak and rated conditions
+                        PeakTotCapTempModFac =
+                            CurveManager::CurveValue(state, simpleWatertoAirHP.TotalCoolCapCurveIndex, ratioTWB, ratioTS, 1.0, 1.0);
+                        RatedTotCapTempModFac =
+                            CurveManager::CurveValue(state, simpleWatertoAirHP.TotalCoolCapCurveIndex, RatedratioTWB, RatedratioTS, 1.0, 1.0);
+                        RatedCoolPowerTempModFac =
+                            CurveManager::CurveValue(state, simpleWatertoAirHP.CoolPowCurveIndex, RatedratioTWB, RatedratioTS, 1.0, 1.0);
+                        // calculate the rated total capacity based on peak conditions
+                        // note: the rated total capacity can be different than the total capacity at
+                        // rated conditions if the capacity curve isn't normalized at the rated
+                        // conditions
+                        if (PeakTotCapTempModFac > 0.0) {
+                            RatedCapCoolTotalDes = CoolCapAtPeak / PeakTotCapTempModFac;
```

Comment:
Calculate the rated capacity based on the load of the coil at peak design conditions and the capacity modifier at peak conditions.